### PR TITLE
Add the ONTOLOGY_HOSTNAME param to the compose file

### LINKED ISSUE #1: ontology-server does not work on localhost
Due to the hardcoded 'server' section in the nginx config. Shall use a docker parameter somehow.

### PR DESCRIPTION
**Pull request by berezovskyi**
Allows to switch between the localhost and the deployment host during
the image build phase.

Closes #1 with help of @likelion.

Signed-off-by: Andrew Berezovskyi <andriib@kth.se>

### CODE REVIEW

File: ontology-server/nginx/Dockerfile [hunk 1]
```diff
@@ -11,6 +13,8 @@ RUN apt-get -yq install nginx
 # configure nginx
 WORKDIR /etc/nginx/sites-available
 COPY cliopatria .
+RUN sed -i 's/%ONTOLOGY_HOSTNAME%/'"$ONTOLOGY_HOSTNAME"'/g' cliopatria
+RUN cat cliopatria
```
Comment (likelion) [inline ontology-server/nginx/Dockerfile]: Maybe we can remove this line later on

**Comment by likelion** [comment]
Overall, I'm happy with the update!

File: ontology-server/nginx/Dockerfile (hunk 1 above)
Comment (berezovskyi) [reply ontology-server/nginx/Dockerfile]: Thanks, commented it out!

**Comment by berezovskyi** [comment]
Build failure due to #6

**Comment by berezovskyi** [comment]
@likelion, `RUN cat cliopatria` is now commented out. Shall we merge this PR?

**Pull request merged**